Google.PowerShell/BigQuery/GcbqTable.cs

```diff
@@ -0,0 +1,198 @@
+    ///   <para>This will return a Table descriptor object for "my_table" in "my_data".</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GcbqTable -Project "my_proj" -Dataset "my_data" "my_table"</code>
+    ///   <para>This returns a Table descriptor object for this project:dataset:table combination.</para>
+    /// </example> 
+    /// <para type="link" uri="(https://cloud.google.com/bigquery/docs/reference/rest/v2/tables)">
+    /// [BigQuery Tables]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "GcbqTable")]
+    public class GetGcbqTable : GcbqCmdlet
+    {
+        /// <summary>
+        /// <para type="description">
+        /// The project to look for datasets in. If not set via PowerShell parameter processing, it will
+        /// default to the Cloud SDK's DefaultProject property.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [ConfigPropertyName(CloudSdkSettings.CommonProperties.Project)]
+        override public string Project { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The ID of the dataset that you want to get a descriptor object for.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [ValidatePattern("[a-zA-Z0-9_]")]
+        public string Dataset { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The ID of the dataset that you want to get a descriptor object for. 
+        /// (This will be used if both InputObject and Dataset are provided)
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipeline = true)]
+        public Dataset InputObject { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The ID of the dataset that you want to get a descriptor object for.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false, Position = 0)]
+        public string Table { get; set; }
+
+        protected override void ProcessRecord()
+        {
+            // Select the DatasetID. Favor InputObject if both present. Default if neither present.
+            if (InputObject == null && Dataset == null)
+            {
+                ThrowTerminatingError(new ErrorRecord(
+                    new Exception("No Dataset Specified"),
+                    "No Dataset Specified",
+                    ErrorCategory.InvalidArgument,
+                    InputObject));
+                return;
+            }
+            Dataset = (InputObject != null) ? InputObject.DatasetReference.DatasetId : Dataset;
+
+            // Create and execute request.
+            if (Table == null)
+            {
+                TablesResource.ListRequest request = new TablesResource.ListRequest(Service, Project, Dataset);
+                var response = request.Execute();
```

Comment:
Ah, yes.  I did that before on the Project cmdlet.  Disregarded it this time because we said to nix the input parameters on the project one.  I've spliced that code in now.